tests/scripts/run_python_model_tests.sh

```diff
@@ -50,4 +50,7 @@ if [ "$ARCH_NAME" != "wormhole_b0" ]; then
 
     # Unet Shallow
     WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -svv models/experimental/functional_unet/tests/test_unet_model.py
+
+    # Llama 3.1 8B single-layer dummy weights tight PCC check
+    WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/wormhole/llama31_8b/tests/test_llama_model.py::test_llama_model_inference[quick]
```

Comment:
Infra doesn't let us submit this, i.e., a test ending with [ ].
We have to use CI fixtures on the test itself. I'll add the change 👍

Reply:
Just pushed a commit for this.